src/components/HelloHell.tsx

```diff
@@ -0,0 +1,8 @@
+export const HelloHell = (): JSX.Element => {
+  return (
+    <>
+      <h1>Hell. I am in HEEEEEEEllll. Hello!!!!</h1>
```

Comment:
This many eeeees is unnecessary, please reduce by 2
```suggestion
      <h1>Hell. I am in HEEEEEllll. Hello!!!!</h1>
```

Reply:
But my computer really thought it was in SEVEN E hell....